main/guides/coreeval/permissions.md

```diff
@@ -0,0 +1,89 @@
+```
+{
+  "consume": {
+    "agoricNamesAdmin": true,
+    "bankManager": true,
+    "board": true,
+    "chainStorage": true,
+    "zoe": "zoe",
+    "feeMintAccess": "zoe",
+    "economicCommitteeCreatorFacet": "economicCommittee",
+    "econCharterKit": true,
+    "provisionPoolStartResult": true,
+    "psmKit": true,
+    "chainTimerService": "timer"
+  },
+  "produce": {
+    "testFirstAnchorKit": true
+  },
+  "installation": {
+    "consume": {
+      "contractGovernor": true,
+      "psm": true,
+      "mintHolder": true
+    }
+  },
+  "instance": {
+    "consume": {
+      "economicCommittee": true
+    }
+  },
+  "issuer": {
+    "produce": {
+      "DAI": true
+    },
+    "consume": {
+      "DAI": true
+    }
+  },
+  "brand": {
+    "consume": {
+      "DAI": true,
+      "IST": true
+    },
+    "produce": {
+      "DAI": true
+    }
+  }
+}
+```
+:::
+
+## Consume Section
+
+In this section you need to set all the permissions that your contract will need to be able to use (i.e., "consume"). Some of the listed permissions in the example PSM permission file above are of general interest to most contracts, while others are more specific to the PSM contract.
+
+* **agoricNamesAdmin**: Grants write access to the Agoric name service. This permission is somewhat specific to the PSM contract.
+* **bankManager**: Grants access to bank-related functionality within ERTP and Zoe. Most contracts will need access.
+* **board**: Grants write access to the board name service. This permission is somewhat specific to the PSM contract.
+* **chainStorage**: Grants write access to the chain storage node, which is required when running `agd query` commands. Thus, most contracts will need access to this.
+* **zoe**: When this permission is set to "zoe", it grqnts access to the Zoe framework. All contracts will need access to this.
+* **feeMintAccess**: When this permission is set to "zoe", the contract will be able to print money. Any contract that mints a currency will need access to this.
+* **economicCommitteeCreatorFacet**, **econCharterKit**, **provisionPoolStartResult**: These 3 permissions are required by governed contracts.
+* **chainTimerService**: When this permission is set to "timer", it grants access to the *chainTimerService*. All governed contracts need access to this so they know when the vote is complete.
+
+## Produce Section
+
+Specifies what, if anything, the contract produces. For example, the example PSM contract produces *testFirstAnchorKit* which is used for testing purposes.
+
+## Installation Section 
+
+Specifies what installations the contract requires. At a minimum, the contract itself should be listed as an installation. Governed contracts should include a *contractGovernor* installation.
+
+## Instance Section
+
+Specifies what instances, if any, the contract produces or consumes.
+
+## Issuer Section
+
+Specifies what tokens, if any, the contract issuer can produce or consume.
+
+## Brand Section
+
```

Comment:
_question_:
How do the `brand section` and `issuer section` differ?

Reply:
I added the following sentence to the Brand Section to clarify: "Note that any brands that the contract may mint are not included in this section, unless they're also produced or consumed in some other capacity."

Reply:
doesn't look like the change made it to this PR yet.

If you're talking about a change you haven't pushed yet, fair enough; I'll stand by.

Reply:
This looks fixed now.

Reply:
It still talks about minting brands.

Also, the brand section is all about sharing the brands created by a contract. I don't think "not included in this section" is on target.

The question was: how are the `brand` section and the `issuer` section different? They're different in that `makeIssuerKit` creates both a brand and an issuer (as well as a mint). The contract creator typically wants to share both of them with prospective clients. The brand is typically shared by producing an item in the `brand` section, and the issuer is shared by producing an item in the `issuer` section. The `permit` here is so that governance can control the namespace.

Reply:
I'm resolving this conversation here, and taking the discussion to Dan's official review below where the same issue is raised.

Reply:
> I'm not seeing it mention minting brands anymore.

ah. I see the fix now. thanks.

Reply:
> unless the Issuer is also named "DAI"?

Yes, as usual, the DAI asset has 3 facets; brand, issuer, and mint. The DAI brand is shared via the `brand` section; the DAI issuer is shared in the `issuer` section, and the DAI mint is held closely rather than being shared.